website/ref/readline-binding.md

```diff
@@ -0,0 +1,19 @@
+<!-- toc -->
+
+# Introduction
+
+The `readline-binding` module provides readline-like key bindings. Such as
```

Comment:
Use a comma instead of period before "such as".

Reply:
Done.